README.md

```diff
@@ -117,3 +117,75 @@ npm publish --access public
 git push
 git push --tags
 ```
+
+
+### Packaging with vercel.pkg
+There are some nuances when packaging with native modues.
+Included native modules are level witch include leveldown and utp-native.
+
+If a module is not set to public then pkg defaults to including it as bytecode.
+To avoid this breaking with the `--no-bytecode` flag we need to add `--public-packages "*"`
+
+#### leveldown
+To get leveldown to work with pkg we need to include the prebuilds with the executable.
+after building with pkg you need to copy from `node_modules/leveldown/prebuilds` -> `path_to_executable/prebuilds`
+You only need to include the prebuild for the arch you are targeting. eg for linux-x64 you need `prebuild/linux-x64`.
+
+The folder structure for the executable should look like this.
+- linux_executable_elf
+- prebuilds
+  - linux-x64
+    - (node files)
+
+#### utp-native
+Including utp-native is simpler, you just need to add it as an asset for pkg.
```

Comment:
Can you explain why `utp-native` is different from `leveldown`?

Reply:
I'm not sure, they include the native bindings almost identically.
leveldown `binding.js`
```
module.exports = require('node-gyp-build')(__dirname)
```
where as utp-native does.
`lib/binding.js`
```
const path = require('path')

module.exports = require('node-gyp-build')(path.join(__dirname, '..'))

```